src/Symfony/Bundle/FrameworkBundle/Command/SecretsSetCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Secrets\SodiumVault;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @author Tobias Schultze <http://tobion.de>
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+final class SecretsSetCommand extends Command
+{
+    protected static $defaultName = 'secrets:set';
+
+    private $vault;
+
+    public function __construct(SodiumVault $vault)
+    {
+        $this->vault = $vault;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDefinition([
+                new InputArgument('name', InputArgument::REQUIRED, 'The name of the secret'),
+            ])
+            ->setDescription('Sets a secret in the storage.')
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> command stores a secret.
+
+    %command.full_name% <name>
```

Comment:
What's about adding an option `-` (like in yaml:lint command) to read secret from STDIN. It would allow multiline secret which is not permitted by QuestionHelper

```
  <info>cat filename | php %command.full_name% <name> -</info>
```

Reply:
the command now accept a 2nd argument which is a file where to read the secret from, `"-"` meaning `STDIN` (`docker create secret` has the same behavior)

Reply:
Should'nt it be documented in the HELP bloc like other commands? (using pipe may be tricky and the hint `reading from STDIN` may not be enough for ppl) 

```
The <info>%command.name%</info> command stores a secret.

You can store secret contents passed from STDIN:

  <info>cat filename | php %command.full_name% <name> -</info>

You can also store secret contained in a file:

  <info>php %command.full_name% <name> filename</info>

Or enter it manually when prompted

  <info>php %command.full_name% <name></info>

```